### DISCUSSION TITLE
Add "dynamic" jobs after a another job is finished

### DISCUSSION THREAD

**Discussion by Walter-Melon** [ksassnowski/venture#70] [Q&A]
Hey,

I have a job that uploads a product via an API to a shop system. The shop handels product images in two steps:
1. Create an empty database image model
2. Upload the file and attach it to the image model

I can create the image model with the initial product upload but now i need to upload the images.
After all images are uploaded i would like to update the product once more to set active for customers.

So the workflow would be:
UploadProduct => UploadImage(1), UploadImage(2), ... => SetProductActive

My plan was to create a job for each image upload and add these jobs to the workflow, how can i do this?

**Comment by stevebauman** [comment]
Hi @Walter-Melon,

My comment here may help you with the implementation of this, as what you describe is not an available feature: https://github.com/ksassnowski/venture/discussions/62#discussioncomment-4256486

**Comment by buddhaCode** [comment]
I have a similar problem. Maybe some knows a workaround or have another approach to the problem?!

1. I retrieve a list of items from an API
2. I create a `n` jobs to fetch the details for each of these items.

@stevebauman I read the comment you mentioned. But I do not see how this might be related to our problem.
